templates/support_channels.html

```diff
@@ -0,0 +1,94 @@
+{% load i18n %}
+{% load static %}
+
+<section class="frontpage frontpage-section">
+	<div class="cards">
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% trans "Documentation" %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://plus.cs.aalto.fi/aplus-manual/master/">{% trans "Aplus manual" %}</a>
```

Comment:
I would like to include the link to the aplus-manual git repo too since people can run it in their own machines: https://github.com/apluslms/aplus-manual

What about the new aplus course template git repo?
https://github.com/apluslms/aplus-course-template

Reply:
I will add a card with the title EN:"Download course templates" FI:"Lataa kurssimallit"